content/en/apps/guides/data/analytics/testing-dbt-models.md

```diff
@@ -0,0 +1,138 @@
+
+- Singular or custom generic data tests for aggregation models:
+For models that perform data aggregation, it is crucial to include singular data tests or custom generic data tests to ensure that the aggregated data meets the expected criteria. These tests help verify that:
+  - Data is accurately aggregated according to the defined logic.
+  - The results align with business expectations and requirements.
+  - Custom data tests are particularly important for aggregation models, as errors in these models can lead to significant discrepancies in reports and analyses.
+
+- Unit Tests for complex logic:
+Unit tests are not strictly required but are highly recommended, especially for models with complex transformation logic. Examples of when to use unit tests include:
+  - Complex SQL Logic: Such as regex, date math, window functions, or extensive CASE WHEN statements.
+  - Custom Calculations: When creating functions or applying unique data processing logic.
+  - Edge Cases: To handle scenarios that are not typically found in actual data but may arise unexpectedly.
+
+## Test implementation in CHT pipeline
+
+Have schema tests for all models and specifically enforcing unique, not_null, and relationships where relevant. (All models should have these tests)
+Add custom data tests for aggregation models to ensure the data is as expected (Only specific models need these tests)
+Add unit tests for complex logic e.g. data math, window functions e.t.c. (not required but good to have especially if model is covered by data tests)
+
+Our project is organized with a `/models` directory containing SQL files and YAML files for generic tests, and a `/test` directory with folders for fixtures and singular tests.
+
+```
+./
+    /models
+        /contacts
+            /tests
+                contacts.yml  --> unit tests
+                persons.yml
+            contact.sql
+            contacts.yml      --> generic data tests
+            patient.sql
+            person.sql
+            place.sql
+        /forms
+        /meta
+        /root
+        /users
+    /test
+        /fixtures                                     --> input data for unit tests
+            data_record/
+                data_record_initial_expected.csv
+                data_record_source_table_initial.csv
+                data_record_document_metadata_initial.csv
+        /sqltest                                       --> singular data tests
+            contact.sql
+            data_record.sql
+            patient.sql
+```
+
+### Data Tests
+
+***Generic data tests*** are included in the model's YAML file under the `data-tests` tag, ensuring basic validations like `relationships`, `not_null`, and `unique`. The following image contains an extract of the properties of the  `contacts.yml` file.
+{{< figure src="contacts-yml.png" link="contacts-yml.png" class=" center col-16 col-lg-12" >}}
```

Comment:
Please replace the image with a code block. This will increase readability, make the page consistent with the rest of the docs site, and support version control changes easier.